src/components/About Us/AboutUs.js

```diff
@@ -0,0 +1,63 @@
+// import React from "react";
+// import "./AboutUs.css";
+
+// function AboutUs() {
+//   return (
+//     <div className="aboutus">
+//       <div className="heading-aboutus"></div>
+//       Hello About Us !
+//     </div>
+//   );
+// }
+
+// export default AboutUs;
+
+import React from "react";
+import "./AboutUs.css";
+import aboutus1 from "../../assets/aboutus1.png";
+import aboutus2 from "../../assets/aboutus2.png";
+import aboutus3 from "../../assets/aboutus3.png";
+
+function AboutUs() {
+  return (
```

Comment:
memoization

Reply:
Actually i am not getting why to use Memoization here for About Us Page

Reply:
Solved